packages/million/dom.ts

```diff
@@ -227,4 +227,13 @@ export const setAttribute = (
   } else {
     removeAttribute$.call(el, name);
   }
+
+  const isInput = el instanceof HTMLInputElement;
+  const isSelect = el instanceof HTMLSelectElement;
+  const isTextArea = el instanceof HTMLTextAreaElement;
+
+  if (name === 'value' && (isInput || isSelect || isTextArea)) {
+    setAttribute$.call(el, name, String(value));
```

Comment:
This is unnecessary, but it can show different values between the `value` attribute and the `value` property. I'm letting it here for consistency and to avoid user confusion.